docs/custom-domain/registrars/cloudflare/cloudflare.md

```diff
@@ -0,0 +1,155 @@
+# How to add a Cloudflare domain to SimpleLogin
+
+## **Add your domain to SimpleLogin**
+
+You have a domain name bought on Cloudflare and you want to use it on SimpleLogin? This documentation is made for you.
```

Comment:
I think we can remove this sentence as this is a documentation and we can use a neutral tone :)